src/application.coffee

```diff
@@ -468,7 +468,7 @@ getEnvAndFormatRemoteApps = (deviceId, remoteApps, uuid, apiKey) ->
 		getEnvironment(app.id, deviceId, apiKey)
 		.then (environment) ->
 			app.environment_variable = environment
-			utils.extendEnvVars(app.environment_variable, uuid)
+			utils.extendEnvVars(app.environment_variable, uuid, app.id)
```

Comment:
It might be worth passing the entire app object, to pave the way for exposing other attributes of the app in future